src/ms.tex

```diff
@@ -496,14 +496,14 @@ \subsection{Numerical Precision, Accuracy, \& Reproducibility}
 \href{https://codata.org/initiatives/data-science-and-stewardship/fundamental-physical-constants/}{\texttt{SI/CODATA
 2018}}. \astropyconstants now defaults to using the \texttt{SI/CODATA 2018}
 values, with the units in \astropyunits based on these constants. Most of
-\astropypkg and affiliate packages build upon \astropyunits, so this update
+\astropy and affiliated packages build upon \astropyunits, so this update
```

Comment:
I changed `astropy` to Astropy because we want to cover the core and Coordinated packages. We can reword if that is too confusing.

Reply:
This reads fine to me.